doc/usage/extensions/autosummary.rst

```diff
@@ -293,16 +293,79 @@ The following variables are available in the templates:
 
 .. data:: members
 
-   List containing names of all members of the module or class.  Only available
+   List containing names of all members of the module or class. Only available
    for modules and classes.
 
 .. data:: inherited_members
 
-   List containing names of all inherited members of class.  Only available for
+   List containing names of all inherited members of class. Only available for
    classes.
 
    .. versionadded:: 1.8.0
 
+.. data:: inherited_qualnames
+
+   List containing the fully qualified names of each inherited member. Will return
+   just the closest parent from which this class was inherited. Only available
+   for classes.
+
+   The following example assumes that this code block has been written as
+   part of a module ``mypackage.test``
+
+   .. code-block:: python
+
+      class foo():
+          foo_attr = "I'm an attribute"
+          def __init__(self):
+              print("object initialised")
+
+          def do_something(self):
+              print("Foo something")
+
+          def _i_am_private(self):
+              print("I'm a private method")
+
+      class bar(foo):
+          def do_something_else(self):
+              print("Bar something")
+
+   Some available parameters for the autosummary of class ``bar`` in the above
```

Comment:
Also add an example for how to use it in a template.

Reply:
Have added an example